Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,107 @@
+//
+//  File.m
+//
+//
+//  Created by Dante Tam on 6/10/21.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "AMPStorage.h"
+
+@interface AMPStorage ()
+@end
+
+@implementation AMPStorage {
+    
+}
+
++ (NSString *)getAppStorageAmpDir {
```

Comment:
Don't declare these functions to be global static. They should belong to a CLASS.

Reply:
Move them into `@implementation AMPStorage`